From 2f12c6652163918658a9db8f258f769c4d7a77c4 Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Mon, 31 Oct 2016 16:59:51 +0800
Subject: [PATCH] video: rockchip: vpu: change the license to GPL v2

The proprietary license would make some kernel reasouces and
function unavailable.

Change-Id: Ibaea3e3389ab05dbd60adfcc0d7e3bba787415c4
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 drivers/video/rockchip/vcodec/vcodec_service.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/vcodec/vcodec_service.c b/drivers/video/rockchip/vcodec/vcodec_service.c
index a92125bd0d59..a1e09b9fa61f 100644
--- a/drivers/video/rockchip/vcodec/vcodec_service.c
+++ b/drivers/video/rockchip/vcodec/vcodec_service.c
@@ -2902,7 +2902,7 @@ static void __exit vcodec_service_exit(void)
 
 module_init(vcodec_service_init);
 module_exit(vcodec_service_exit);
-MODULE_LICENSE("Proprietary");
+MODULE_LICENSE("GPL v2");
 
 #ifdef CONFIG_DEBUG_FS
 #include <linux/seq_file.h>
-- 
2.35.3

